nipype/pipeline/engine/nodes.py

```diff
@@ -577,25 +577,25 @@ def _run_command(self, execute, copyfiles=True):
             self._originputs = deepcopy(self._interface.inputs)
             self._copyfiles_to_wd(execute=execute)
 
-        message = '[Node] Running "%s" ("%s.%s")'
+        message = '[Node] Running "{}" ("{}.{}")'
         if issubclass(self._interface.__class__, CommandLine):
             try:
                 cmd = self._interface.cmdline
             except Exception as msg:
-                result.runtime.stderr = '%s\n\n%s' % (
+                result.runtime.stderr = '{}\n\n{}'.format(
                     getattr(result.runtime, 'stderr', ''), msg)
                 _save_resultfile(result, outdir, self.name)
                 raise
             cmdfile = op.join(outdir, 'command.txt')
             with open(cmdfile, 'wt') as fd:
                 print(cmd + "\n", file=fd)
-            message += ', a CommandLine Interface with command:\n%s' % cmd
-        logger.info(message, self.name, self._interface.__module__,
-                    self._interface.__class__.__name__)
+            message += ', a CommandLine Interface with command:\n{}'.format(cmd)
```

Comment:
why not `cmd.replace('%', '%%')`?

Reply:
Actually, this problem now will happen the same if the command has braces...

Reply:
good call - I would prefer to not alter the `cmd` if possible, what about substitution first?

Reply:
`cmd.replace('%', '%%')` is not an in-place operation, why not just do it here?

Reply:
sure, we can replace to cover against substitution (`%%`) / escape characters (`\", \\`, etc), but isn't `.format` cleaner?

Reply:
It is just to use python's logging standards, but I'm not particularly attached to one solution or another 👍